src/Components/Missions.js

```diff
@@ -1,8 +1,36 @@
 import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { joinMission } from '../redux/missions/missions';
 
 const Missions = () => {
+  const missions = useSelector((state) => state.missions);
+  const dispatch = useDispatch();
   return (
-    <div>ERROR!! Page Not Found</div>
+    <div className="missions_container">
+      <table>
+        <tbody>
+          <tr>
+            <th>Mission</th>
+            <th>Description</th>
+            <th>Status</th>
+            <th />
+          </tr>
+          {missions &&  missions.map((mission) => (
+          <tr key={mission.mission_id}>
+            <td className="missionName">{mission.mission_name}</td>
+            <td className="missionDescription">{mission.description}</td>
+            <td className={`userBadge ${mission.reserved ? 'member' : ' '}`}><label>
+              {mission.reserved ? 'Active member' : 'NOT A MEMBER'}</label></td>
```

Comment:
Semantic HTML: you are using a label element to display the membership status. However, label elements are typically used for form input descriptions. Consider replacing this with a more semantic HTML element such as p or span.